src/release_notes/factory/record_factory.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from github_integration.model.commit import Commit
+from github_integration.model.issue import Issue
+from github_integration.model.pull_request import PullRequest
+from release_notes.model.record import Record
+
+
+class RecordFactory:
```

Comment:
why having the package `factory` that has only 1 file in there? Also same for formatter - do you anticipate more files, i.e. more growth in this package?

Reply:
You are right. Removed the packages until there was a real need. See commit a46aa1fc908d1188f297c2281ced67a444dc3eb7.